packages/main/src/MenuItem.ts

```diff
@@ -96,6 +97,32 @@ class MenuItem extends UI5Element {
 	@property({ type: Boolean })
 	disabled!: boolean;
 
+	/**
+	 * Defines if a loading indicator would be displayed inside the corresponding sub-menu popover.
+	 *
+	 * Note: If set to <code>true</code> a <code>ui5-busy-indicator</code> component will be displayed into the related to the current <code>ui5-menu-item</code> sub-menu popover.
```

Comment:
If set to <code>true</code> a <code>ui5-busy-indicator</code> component will be displayed into the related one to the current <code>ui5-menu-item</code> sub-menu popover.

Reply:
done